demisto_sdk/tests/workflow_test.py

```diff
@@ -202,6 +202,7 @@ def function_setup():
         content_git_repo = ContentGitRepo()
     # Function setup
     content_git_repo.git_cleanup()
+    content_git_repo.run_command("npm install")
```

Comment:
Doesn't this mean that for every test we re-install npm packages?

Reply:
yes I think so. But we're also reinstalling the repo so I think we need to

Reply:
if the repo was taken from cache and these arent installed it will finish quickly